assets/js/components/settings/SettingsActiveModules.js

```diff
@@ -16,62 +16,62 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
 /**
  * WordPress dependencies
  */
-import { useState } from '@wordpress/element';
+import { useState, useCallback } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { clearWebStorage } from '../../util';
-import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
 import Data from 'googlesitekit-data';
+import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import { clearWebStorage } from '../../util';
 import Layout from '../layout/Layout';
-import SettingsModule from './settings-module';
+import SettingsActiveModule from './SettingsActiveModule';
 const { useDispatch, useSelect } = Data;
 
-const SettingsActiveModules = ( { activeModule, moduleState, setModuleState } ) => {
-	const { submitChanges } = useDispatch( CORE_MODULES );
-	const [ error, setError ] = useState( false );
+export default function SettingsActiveModules( { activeModule, moduleState, setModuleState } ) {
+	const [ error, setError ] = useState( undefined );
 	const [ isSaving, setIsSaving ] = useState( false );
+
+	const { submitChanges } = useDispatch( CORE_MODULES );
 	const modules = useSelect( ( select ) => select( CORE_MODULES ).getModules() );
 
-	const onEdit = ( slug ) => {
+	const onEdit = useCallback( ( { slug } ) => {
 		setModuleState( slug, 'edit' );
-	};
+	}, [] );
 
-	const onCancel = ( slug ) => {
+	const onCancel = useCallback( ( { slug } ) => {
 		setModuleState( slug, 'view' );
-	};
+	}, [] );
```

Comment:
`setModuleState` comes from props so let's be sure to include it as a dependency, even though it shouldn't ever change.

Reply:
Updated.